t2.lic

```diff
@@ -122,9 +123,9 @@ before_dying do
     stop_script(script_name) if Script.running?(script_name)
   end
   # Don't bother releasing cyclic for NMU guilds
-  fput('release cyclic') unless %w[Commoner Barbarian Thief].include?(DRStats.guild)
+  DRCA.release_cyclics unless %w[Commoner Barbarian Thief].include?(DRStats.guild)
```

Comment:
Maybe not part of this PR, but _if_ we feel this check is necessary, I think it should be moved to a guard clause within `release_cyclics` itself.

Reply:
We're planning on touching the function in common-arcana anyway in the near future, to make not releasing some cyclics optional. It would make sense to add the check there at that time, if you're ok with that idea.

Reply:
Sounds great